tests/src/common/Wsk.scala

```diff
@@ -202,13 +202,14 @@ trait HasActivation {
     }
 }
 
-class WskAction()
+class WskAction(override val usePythonCLI: Boolean = true) 
```

Comment:
@csantanapr we should document the git commit hook with runs scan code for devs to adopt.